client/src/components/ChatSideBanner/Chat.js/Chat.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState, useCallback } from 'react';
+import { Box, Badge, Avatar, Typography, Divider } from '@material-ui/core';
+import { useConversation } from '../../../context/useActiveConvoContext';
+import useStyles from './useStyles';
+import { getConversations } from '../../../helpers/APICalls/conversation';
+import Loading from '../../Layout/Loading';
+
+const Chat = () => {
+  const classes = useStyles();
+  const [conversations, setConversations] = useState([]);
+  const { updateActiveConversation } = useConversation();
+
+  useEffect(() => {
+    getConversations().then((conversations) => {
+      setConversations(conversations);
+    });
+  }, []);
+
+  const setActiveConvo = useCallback(
+    (id: any) => {
+      const activeConvo = conversations.find((convo: any) => {
+        return convo._id === id;
+      });
+
+      updateActiveConversation(activeConvo);
+    },
+    [updateActiveConversation, conversations],
+  );
+
+  return (
+    <>
+      {conversations.length === 0 ? (
+        <Loading />
```

Comment:
Based on the ternary operator if we don't have any conversations, won't this render a persistent loading spinner?